### PR TITLE
Updating code as per pep8 recommendations

### PR DESCRIPTION
**Pull request by shrayasr**
Not everything is updated, some parts don't make sense enough to follow.

These are the few more errors pending: 

```
(env)shrayas at shrayas-mac in ~/code/hacks/broadgauge on master*
❯ pep8 broadgauge/
broadgauge/oauth.py:26:52: E231 missing whitespace after ':'
broadgauge/oauth.py:27:33: E225 missing whitespace around operator
broadgauge/oauth.py:28:55: E231 missing whitespace after ':'
broadgauge/oauth.py:29:33: E225 missing whitespace around operator
broadgauge/oauth.py:68:52: E231 missing whitespace after ':'
broadgauge/oauth.py:69:33: E225 missing whitespace around operator
broadgauge/oauth.py:70:55: E231 missing whitespace after ':'
broadgauge/oauth.py:71:33: E225 missing whitespace around operator
broadgauge/oauth.py:72:47: E231 missing whitespace after ':'
broadgauge/oauth.py:72:68: E225 missing whitespace around operator
broadgauge/oauth.py:73:41: E225 missing whitespace around operator
broadgauge/webapp.py:20:80: E501 line too long (80 > 79 characters)
```

In `oauth.py`: The `missing whitespace after ':'` error occurs because there isn't a whitespace after `:` when `:` is being used within `https://`. Which doesn't make sense. 

In `webapp.py`: Line 20 is a line describing a URL pattern. Doesn't make sense to split that to another line.

---

This is the first time i've found out about the `pep8` tool and done these changes. Please advise.

### CODE REVIEW

**Comment by kracekumar** [comment]
Agreed those two can be ignored.

**Comment by shrayasr** [comment]
Sounds good. Apart from that everything else is taken care of (I believe).

File: broadgauge/default_settings.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 github_client_id = 'f6adcb4752f5c0738aa4'
 github_client_secret = '4f8b629504f059001c8d823f1710b30c73333354'
 
-google_client_id = '360651067559-f04a4e5scoa0bgau2tar06q05jeg5j2n.apps.googleusercontent.com'
+google_client_id =
+'360651067559-f04a4e5scoa0bgau2tar06q05jeg5j2n.apps.googleusercontent.com'
```
Comment (anandology) [inline broadgauge/default_settings.py:10]: Isn't this an error?

File: broadgauge/oauth.py [hunk 2]
```diff
@@ -7,23 +7,27 @@
...
-            base_url='https://api.github.com/')
+        OAuth2Service.__init__(self,
+                               client_id=web.config.github_client_id,
+                               client_secret=web.config.github_client_secret,
+                               name='github',
+                               authorize_url='https://github.com
+                               /login/oauth/authorize',
+                               access_token_url='https://github.com
+                               /login/oauth/access_token',
+                               base_url='https://api.github.com/')
```
Comment (anandology) [inline broadgauge/oauth.py:30]: What was a need for changing this? I'm sure adding whitespace in URLs is not a good idea.

File: broadgauge/oauth.py [hunk 3]
```diff
@@ -56,12 +62,15 @@ class Google(OAuth2Service):
...
-            base_url='https://www.googleapis.com/oauth2/v1/')
+                               client_id=web.config.google_client_id,
+                               client_secret=web.config.google_client_secret,
+                               name='google',
+                               authorize_url='https://accounts.google.com
+                               /o/oauth2/auth',
+                               access_token_url='https://accounts.google.com
+                               /o/oauth2/token',
+                               base_url='https://www.googleapis.com/
+                               oauth2/v1/')
```
Comment (anandology) [inline broadgauge/oauth.py:73]: Same as above.

File: broadgauge/webapp.py [hunk 4]
```diff
@@ -155,11 +164,14 @@ def find_user(self, email):
         return None
 
     def signup(self, i, userdata):
-        user = User.find(email=userdata['email']) or User.new(name=userdata['name'], email=userdata['email'])
-        org = Organization.new(name=i.name, city=i.city, admin_user=user, role=i.role)
+        user = User.find(email=userdata['email'])
+        or User.new(name=userdata['name'], email=userdata['email'])
+        org = Organization.new(name=i.name, city=i.city, admin_user=user,
+                               role=i.role)
```
Comment (anandology) [inline broadgauge/webapp.py:170]: Sorry, this'll be syntax error again.

**Comment by anandology** [comment]
@shrayasr Thanks for try to fix that, but it introduced couple of syntax errors and broken some URLs. Could you please fix that. 

Looks like your emacs is not smart enough.

**Comment by shrayasr** [comment]
Didn't actually try to run anything. I'll set up the dB today and test it
against that and make changes. Apologies. I'll fix this today and update
the PR.

PS: vim :D

File: broadgauge/oauth.py (hunk 2 above)
Comment (shrayasr) [inline broadgauge/oauth.py:30]: So PEP8 tells me that those lines are under-indented. Should I just fix the URL moving to the next line or shall I revert this entire change in itself. The previous version didn't have the URL exceeding the 80 col mark but it was under indented.

**Comment by shrayasr** [comment]
Would also be a good idea to try and get the unit tests out asap so changes like this can be tested more easily. 

It was wrong on my part to push changes without testing them properly. Sorry.

**Comment by anandology** [comment]
Merged the relevant part of this pull request. e065f6919dd840b543aff155bb727d0b6efbc2e5

Thanks @shrayasr.

**Comment by shrayasr** [comment]
Really nice of you to take time to do the partial merge. Thank you :)

**Comment by anandology** [comment]
basic unit test is added and integrated with Travis CI. It just imports all the code and detects syntax errors.

**Comment by shrayasr** [comment]
Super! Should be enough for now

**Pull request closed without merging**